docs/release_notes/clearml_server/enterprise/ver_3_26.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Version 3.26
+---
+
+### Enterprise Server 3.26.0
+
+:::note Application Gateway Compatibility 
+If you are using the ClearML Application Gateway to run session applications, make sure to update the gateway components to the following versions:
```

Comment:
_to run interactive/GPUaaS applications_

Reply:
make sure to update the gateway components to the following versions or upgrade the App Gateway helm chart to the following version